components/Layout.tsx

```diff
@@ -4,15 +4,22 @@ import Navbar from "./navigation/Navbar";
 
 type LayoutProps = {
   children: ReactNode;
+  defaultLayout: String;
 };
 
-const Layout = ({ children }: LayoutProps) => {
+const Layout = ({ children, defaultLayout }: LayoutProps) => {
```

Comment:
We should maybe remove this for now as it's not worth having a property that just affects one page in a wrapper component that's related to all pages in my opinion.